app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,61 @@
+class PostsController < ApplicationController
+
+
+    before_action :set_post, only: [:show, :edit, :update, :delete, :destroy]
+    before_action :authenticate_user!
+    
+    def index
+        #@posts = Post.all.order("created_at DESC")
```

Comment:
Commented code should never be committed. You seem to have forgotten this one here. Please remove!